tests/fixtures/parser/ftl/errors/09-block-expr.ftl

```diff
@@ -0,0 +1,7 @@
+key = Test { $foo ->
+  *[other] Value
+}
+
+key = { $foo ->
+  *[other] Value
+} Test
```

Comment:
If I'm reading the code right, this second `key` is actually successfully parsed up to the newline after `  *[other] Value` and then an error is produced for the remaining `} Test`. I find this behavior surprising; I'd expect the whole message to be treated as Junk.